ports/openvr/portfile.cmake

```
@@ -1,15 +1,14 @@
 vcpkg_fail_port_install(ON_ARCH "arm" ON_TARGET "uwp" "osx")
+set(VCPKG_LIBRARY_LINKAGE dynamic)
```

Ports should never directly set library linkage; they can instead use `vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY)`. (The existing port is also incorrect).

---

That's what I did initially, but it failed to build with the x64-windows-static triplet, because of the static CRT : 
```
  Refusing to build unexpected dynamic library against the static CRT.  If
  this is desired, please configure your triplet to directly request this
  configuration.
```
Is it better to let the build fail in this case?

---

Yep, this case (mixing dynamic libraries with static CRT) can cause tons of problems for downstream consumers if they aren't expecting it. This is why the helper is good -- it does the right thing :)